src/equations/inviscid_burgers_1d.jl

```diff
@@ -62,7 +62,7 @@ Source terms used for convergence tests in combination with
   L = 1
   f = 1/L
   omega = 2 * pi * f
-  du = omega * cos(omega * (x[1] - t)) * (1 + sin(omega * (x[1] - t)))
+  du = omega * A * cos(omega * (x[1] - t)) * (1 + A * sin(omega * (x[1] - t)))
```

Comment:
```suggestion
  du = omega * A * cos(omega * (x[1] - t)) * (c - 1 + A * sin(omega * (x[1] - t)))
```
I think this should be required to make it work with general `c`.

Reply:
Ah yes - this also relieves my confusion about the seemingly different signs of $\partial_t u$ and $\partial_x 0.5 u^2$.

Reply:
I would prefer `-1 + c + A * sin(omega * (x[1] - t))`  but this is personal preference I guess.